src/config/2019.json

```
@@ -2,7 +2,7 @@
   "settings": [
     {
       "is_live": true,
-      "supported_languages": ["en","es","fr","ja"],
+      "supported_languages": ["en","es","fr","ja","zh-CN"],
```

I got this wrong. Should be this:
```suggestion
      "supported_languages": ["en","es","fr","ja","zh_CN"],
```